src/utils.ts

```diff
@@ -82,6 +82,12 @@ export type NavigationBarIcon = Readonly<{
      * These properties will be merged to the tracking event produced by the native side
      */
     trackingProperties?: Record<string, string>;
+
+    /**
+     * Identifies the action as opening a section subject to maintenance mode rules, and which section it is.
+     * If Non null it may be that the native app will show a maintenance overlay instead of the usual behaviour when clicked.
+     */
+    maintenanceSectionName?: string;
```

Comment:
after reading this, I still don't know what this does. Please can you link a spec about this or something?

Reply:
Original conversation

https://teams.microsoft.com/l/message/19:5e98d1145be74c9290e67741e01c0100@thread.skype/1717081990110?tenantId=9744600e-3e04-492e-baa1-25ec245c6f10&groupId=bef9be07-03e4-4afb-8a79-f720a63e4c28&parentMessageId=1717081990110&teamName=Novum&channelName=Apps%20Team-public&createdTime=1717081990110

Ticket: https://jira.tid.es/browse/ACCOUNT-25673

Reply:
> If Non null it may be that the native app will show a maintenance overlay instead of the usual behaviour when clicked.

This sentence is quite ambiguous